lua/nvim-lsp-installer/servers/hls/install.mjs

```diff
@@ -22,7 +22,9 @@ await $`rm hls.tar.gz`;
 await $`chmod +x haskell*`;
 
 const scriptContent = `#!/usr/bin/env bash
-PATH="$PATH:${__dirname}" "${__dirname}/haskell-language-server-wrapper" --lsp`;
```

Comment:
Ah! I think this was originally supposed to be `process.cwd()` instead of the `__dirname` of the script itself